src/core/Toast/Toast.test.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Toast } from './Toast';
+import { axeTest } from '../../utils/test';
+
+describe('props', () => {
+  describe('headingText', () => {
+    const ToastWithElementChild = (
+      <Toast headingText="Test heading">Test Toast</Toast>
+    );
+    const { container } = render(ToastWithElementChild);
+    expect(container.firstChild).toHaveTextContent('Test heading');
+  });
+  describe('headingVariant', () => {
+    const { getByRole } = render(
+      <Toast headingText="Test heading" headingVariant="h3">
+        Test Toast
+      </Toast>,
+    );
+    const heading = getByRole('heading', { level: 3 });
+    expect(heading).toHaveClass('fi-heading--h3');
+  });
+  describe('id', () => {
+    const ToastWithElementId = <Toast id="TestId">Test Toast</Toast>;
+    const { container } = render(ToastWithElementId);
+    expect(container.querySelector('#TestId')).toHaveClass(
+      'fi-toast-content-wrapper',
+    );
```

Comment:
I suggest to look how the `id` prop is checked in the RadioButton. Currently this tests expects certain class instead of the id.

https://github.com/vrk-kpa/suomifi-ui-components/blob/340e7a3c0703308020e3dc38c59f4c5b5fb5e23e/src/core/Form/RadioButton/RadioButton.test.tsx#L118-L124

Reply:
I could get better way to get this id test working.